service_mesh/service_mesh_day_two/ossm-auto-route.adoc

```diff
@@ -0,0 +1,69 @@
+[id="ossm-auto-route"]
+= Automatic route creation
+include::modules/ossm-document-attributes.adoc[]
+toc::[]
+
+{product-title} routes for Istio Gateways are automatically managed in {ProductName}. Every time an Istio Gateway is created, updated or deleted inside the service mesh, an {product-title} route is created, updated or deleted. 
+
+[id="ossm-auto-route-enable_{context}"]
+== Enabling Automatic Route Creation
+A {ProductName} control plane component called Istio {product-title} Routing (IOR) synchronizes the gateway route. You can deploy it as part of the control plane in the {ProductName} installation process, but you must enable it first.
+
+. In the `ServiceMeshControlPlane` resource, add the `ior_enabled` parameter and set it to `true`. For example, see the following resource snippet:
+
+----
+gateways:
+ istio-egressgateway:
+   autoscaleEnabled: false
+   autoscaleMin: 1
+   autoscaleMax: 5
+ istio-ingressgateway:
+   autoscaleEnabled: false
+   autoscaleMin: 1
+   autoscaleMax: 5
+   ior_enabled: true
+----
+
+For more information, see xref:../../service_mesh/service_mesh_install/customizing-installation-ossm.adoc#ossm-cr-gateway_customizing-installation-ossm[Istio gateway configuration].
+
+[id="ossm-auto-route-subdomains_{context}"]
+== Subdomains
+
+{ProductName} creates the route with the subdomain, but {product-title} must be configured to enable it. Subdomains, for example, `*.domain.com` are supported but not by default. Cluster administrators can refer to the {product-title} documentation for instructions on how to enable subdomains.
+
+For example, if the following gateway is created:
+
+----
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: gateway1
+spec:
+  selector:
+    istio: ingressgateway
+  servers:
+  - port:
+      number: 80
+      name: http
+      protocol: HTTP
+    hosts:
+    - www.bookinfo.com
+    - bookinfo.example.com
+----
+
+Then, the following {product-title} Routes are created automatically. You can check that the routes are created with the following command.
+
+----
+$ oc -n <your-control-plane-namespace> get routes
+NAME              HOST/PORT                                            PATH      SERVICES               PORT      TERMINATION   WILDCARD
```

Comment:
Perhaps you could get rid of those spaces between the columns so that we don't have a horizontal scrollbar in the rendered doc. They are not relevant for the sake of the content.